bin/learn-test

```diff
@@ -45,6 +45,10 @@ OptionParser.new do |opts|
     options[:keep] = true
   end
 
+  opts.on('--fail-fast', 'Stop running test suite on first failed test') do |f|
```

Comment:
@loganhasson i just reused language from [rspec docs](https://relishapp.com/rspec/rspec-core/docs/command-line/fail-fast-option) here. lemme know if you wanna change it